source/code/go/src/plugins/glide.yaml

```diff
@@ -1,10 +1,8 @@
-package: plugins
+package: .
 import:
 - package: github.com/fluent/fluent-bit-go
   subpackages:
   - output
-- package: github.com/mitchellh/mapstructure
```

Comment:
i thought this was already removed? No ?

Reply:
Removed dependecy from the code only. Wasnt removed from glide